config/crds/all-crds.yaml

```diff
@@ -2779,3 +2779,1039 @@ status:
+                  like by-id type or by-path type link
+                properties:
+                  kind:
+                    description: Kind is the type of link like by-id or by-path.
+                    type: string
+                  links:
+                    description: Links are the soft links of Type type
+                    items:
+                      type: string
+                    type: array
+                type: object
+              type: array
+            filesystem:
+              description: FileSystem contains mountpoint and filesystem type
+              properties:
+                fsType:
+                  description: Type represents the FileSystem type of the block device
+                  type: string
+                mountPoint:
+                  description: MountPoint represents the mountpoint of the block device.
+                  type: string
+              type: object
+            nodeAttributes:
+              description: NodeAttributes has the details of the node on which BD
+                is attached
+              properties:
+                nodeName:
+                  description: NodeName is the name of the Kubernetes node resource
+                    on which the device is attached
+                  type: string
+              required:
+              - nodeName
+              type: object
+            parentDevice:
+              description: "ParentDevice was intented to store the UUID of the parent
+                Block Device as is the case for partitioned block devices. \n For
+                example: /dev/sda is the parent for /dev/sdap1 TODO @kmova to be implemented/deprecated"
+              type: string
+            partitioned:
+              description: Partitioned represents if BlockDevice has partions or not
+                (YES/NO) Currently always default to NO. TODO @kmova to be implemented/deprecated
+              type: string
+            path:
+              description: Path contain devpath (e.g. /dev/sdb)
+              type: string
+          required:
+          - capacity
+          - details
+          - devlinks
+          - nodeAttributes
+          - partitioned
+          - path
+          type: object
+        status:
+          description: DeviceStatus defines the observed state of BlockDevice
+          properties:
+            claimState:
+              description: claim state of the block device
+              type: string
+            state:
+              description: State is the current state of the blockdevice (Active/Inactive)
+              type: string
+          required:
+          - claimState
+          - state
+          type: object
+      type: object
+  version: v1alpha1
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
@prateekpandey14 mentioned in https://github.com/openebs/openebs/issues/3052#issuecomment-719892488 that Helm charts anyway get a refactor, so _in some way_ you'll need to automatically extract YAML CRDs from Go code, e.g. to put them into the Helm charts. Mind _this PR_ is a beginner change without any knowledge of OpenEBS code or repo structure. Since you want to split into different Helm charts, a single `all-crds.yaml` file may not be useful after all... you can treat this PR as idea.